### PR TITLE
LPD-30006 - Refactor LocalFile.InstanceSettings#EditDefaultUserAssociationsUserGroups into a PLaywright test

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-30006

### CODE REVIEW

File: modules/test/playwright/tests/users-admin-web/defaultUserAssociations.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+test('LPD-30006 edit configuration users default user associations CP', async ({
+	apiHelpers,
+	defaultUserAssociationsPage,
+	editUserPage,
+	instanceSettingsPage,
+	page,
+	usersAndOrganizationsPage,
+}) => {
+	const userGroup = await apiHelpers.headlessAdminUser.postUserGroup();
```
Comment (smottal) [inline modules/test/playwright/tests/users-admin-web/defaultUserAssociations.spec.ts:31]: we are not deleting this after the test.
I think the best solution is to add the ```if (this.apiHelpers instanceof DataApiHelpers) {...```  block in the `postUserGroup` method (have a look to the previous method on how it's implemented) and the deletion of it (inside the `apiHelper`)

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by matyaswollner** [comment]
ci:test:sf

**Review by smottal — APPROVED** [review]

**Comment by matyaswollner** [comment]
ci:test:sf

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:playwright:nocompile

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**